R/load_genomic_breaks.R

```diff
@@ -78,17 +78,29 @@ load_genomic_breaks_GFF <- function (
   type = "match_part")
 {
   gb <- import.gff3(file)
-  if (! is.null(target_bsgenome))
-    gb <- GRanges(gb, seqinfo = seqinfo(target_bsgenome))
+  if (!is.null(target_bsgenome)) {
+    if (is.character(target_bsgenome)) {
+      target_seqinfo <- seqinfo(readDNAStringSet(target_bsgenome))
+    } else {
+      target_seqinfo <- seqinfo(target_bsgenome)
+    }
+    gb <- GRanges(gb, seqinfo = target_seqinfo)
+  }
   # Discard cross_genome_match parent (used for block display in Zenbu)
   gb <- gb[gb$type == type]
   # Discard unused information
   gb$phase <- gb$Parent <- gb$Target <- gb$ID <- gb$source <- gb$type <- NULL
   # Convert query coordinates to GRanges
-  if (! is.null(query_bsgenome))
-    gb$query <- GRanges(gb$Name, seqinfo = seqinfo(query_bsgenome))
-  else
+  if (!is.null(query_bsgenome)) {
+    if (is.character(query_bsgenome)) {
+      query_seqinfo <- Seqinfo(readDNAStringSet(query_bsgenome))
```

Comment:
Should be `seqinfo` instead of `Seqinfo`

Reply:
Typo is fixed in the next commit